check_python.sh

```diff
@@ -142,8 +143,9 @@ pushd "${PROJECT_REPO}" > /dev/null || exit 1
         fi
       fi
     else
-      if grep '^#!/bin/.*python' "${script_file}"; then
-        if grep '^#!/bin/.*python3' "${script_file}"; then
+      if (head -1 "$script_file" | grep -q -E '^#!(/usr)?/bin/.*python'); then
+        if (head -1 "$script_file" | \
+            grep -q -E '^#!(/usr)?/bin/.*python3'); then
```

Comment:
Then this could be simplified as:

```suggestion
        if (head -1 "$script_file" | \
            grep -q -e 'python3'; then
```